source/hello.vb

```diff
@@ -0,0 +1,6 @@
+Module HelloWorld
+    Sub Main()
+        Console.WriteLine("Hello, World!")
+        Console.ReadLine()
```

Comment:
When you run this program, it will print "Hello, World!" to the console. The `Console.ReadLine()` at the end ensures the console window stays open until you press Enter.